src/schema/ecommerce/types/order_line_item.ts

```diff
@@ -20,13 +21,26 @@ export const OrderLineItemType = new GraphQLObjectType({
     artwork: {
       type: Artwork.type,
       description: "Artwork that is being ordered",
+      deprecationReason:
```

Comment:
im not super sure about deprecating the `artwork` field since `artwork_id` is still a thing being returned from Exchange schema and there may be some usages on exposing it. I agree we should encourage people using `artworkVersion` though but I think `artwork` may still have its own use.
also think of the case where we want to know for example if the current version is the latest version, we'd have to fetch `artwork` and check latest version. This could use some helper field too `is_latest_version` type of thing but that can be added when needed.

Reply:
Hmm, good point! I'll take off the deprecation. 👍